content/post/know-your-aws-community-day-india-2020-speaker-abesh-rajasekharan.md

```diff
@@ -0,0 +1,21 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T16:54:08Z
+description = "Abesh Rajasekharan will be speaking on \"Using Ansible to manage Hybrid Architecture\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/abesh.PNG"
+tags = ["AWS", "AWSCommunityDay2020", "Speaker", "Ansible"]
+title = "Know Your AWS Community Day India 2020 Speaker - Abesh Rajasekharan"
+
++++
+Abesh Rajasekharan is working as a Senior Engineering Manager at Thomson Reuters will be speaking on "**Using Ansible to manage Hybrid Architecture**" from 02:00 PM - 02:45 PM Track 2 on 16th October 2020 AWS Community Day India 2020.
```

Comment:
>  working as [...] will be speaking on 


is not grammatically right. This should be 

`<name>` is currently `<title>` at `<company>`. `<First name>` will be speaking on <session title>....

Reply:
Corrected.